nasa/harmony
# HARMONY-1895: Add labels to jobs listings and status pages

## Jira Issue ID
HARMONY-1895

## Description
Adds labels to jobs listings and status pages

## Local Test Steps
* run a job and pass in labels using the `label` paramater
* load the jobs listing page (may not show any jobs on localhost due to a bug Chris discovered) and verify that labels are given for the jobs
http://localhost:3000/jobs
* load the admin jobs listing page and verify labels are shown
http://localhost:3000/admin/jobs
* load the job status page and verify labels are shown
http://localhost:3000/jobs/<job ID>
* load the admin job status page and verify labels are shown
http://localhost:3000/admin/jobs<job ID>

## PR Acceptance Checklist
